src/buf.yaml

```diff
@@ -1,31 +0,0 @@
-# Copyright Materialize, Inc. and contributors. All rights reserved.
-#
-# Use of this software is governed by the Business Source License
-# included in the LICENSE file at the root of this repository.
-#
-# As of the Change Date specified in that file, in accordance with
-# the Business Source License, use of this software will be governed
-# by the Apache License, Version 2.0.
-
-version: v1
-breaking:
-  use:
-    - WIRE
-  ignore:
-    # does currently not require backward-compatibility
-    - cluster-client
-    # does currently not require backward-compatibility
-    - compute-client
-    # does currently not require backward-compatibility
-    - storage-client
-    # does not currently require backward-compatibility
-    - stash
-    # still under active development
-    - persist-types
-lint:
-  use:
-    - DEFAULT
-build:
-  excludes:
-    # external code
-    - testdrive/ci/protobuf-install
```

Comment:
I think we still want to keep the other config options and only remove the `ignore` section?

Reply:
oops. fixed